Mathlib/MeasureTheory/Function/ConditionalExpectation/Basic.lean

```diff
@@ -420,4 +420,10 @@ theorem tendsto_condexp_unique (fs gs : ℕ → α → F') (f g : α → F')
   exact tendsto_nhds_unique_of_eventuallyEq hcond_gs hcond_fs (eventually_of_forall hn_eq)
 #align measure_theory.tendsto_condexp_unique MeasureTheory.tendsto_condexp_unique
 
+/-- Total probability law using `condexp` as conditional probability. -/
+theorem integral_condexp_indicator [mF : MeasurableSpace F] {Y : α → F} (hY : Measurable Y)
+    [SigmaFinite (μ.trim hY.comap_le)] {A : Set α} (hA : MeasurableSet A) :
+    ∫ x, (μ[(A.indicator fun _ ↦ (1 : ℝ)) | mF.comap Y]) x ∂μ = (μ A).toReal := by
+  rw [integral_condexp, integral_indicator hA, setIntegral_const, smul_eq_mul, mul_one]
```

Comment:
Can you put this right after `integral_condexp`? Since it's a consequence of that lemma, they should go together.